modes/basic-dev-mode/src/toolbarButtons.js

```diff
@@ -54,147 +23,91 @@ function _createWwwcPreset(preset, title, subtitle) {
   };
 }
 
-const toolbarButtons = [
-  // Measurement
+function _createSetToolActiveCommands(toolName, toolGroupIds = ['default', 'mpr', ]) {
```

Comment:
This feels like the options to the toolbars should include the toolbar item being selected, and that the tool group ids should belong to a set of tool groups being defined, so that you get 1 setToolActive call, and it is completely generic across all tools - that is, something like:

const commands = [{ commandName: 'setToolActive' }];

You might need to add the context here as well, but if the command caller for the tool group options automatically includes the tool group ids to default to, plus the commandName itself into the comman options, then it can automatically iterate/apply with a shared command.